Posit Infrastructure/How to Install and Use Geospatial R Packages.md

```diff
@@ -175,3 +176,74 @@ Then, geospatial R packages can be loaded in the same way as normal with calls t
 ```r
 library(terra)
 ```
+
+## Adding code to your .Rprofile
+
+You can add code to your personal `.Rprofile` or create and add code to a project's file. Adding code to your personal `.Rprofile` will work *for you* but no one else, adding it to a project's `.Rprofile` will work *for anyone* loading the project but this will need to be done for every project (that uses geospatial packages).
+
+You can use the [`{usethis}`](https://usethis.r-lib.org/) package to easily open the correct file.
+-  `usethis::edit_r_profile("user")` will open your personal `.Rprofile` for editing.
+-  `usethis::edit_r_profile("project")` will open the current project's  `.Rprofile` for editing (and will error if you're not in a project).
+
+You should add all of the below code (comments are optional).
+
+```r
+# Set environment variables to point to installations of geospatial libraries ----
+
+# Append paths to GDAL and PROJ to 'LD_LIBRARY_PATH'
+Sys.setenv(
+  LD_LIBRARY_PATH = paste(
+    Sys.getenv("LD_LIBRARY_PATH"),
+    "/usr/gdal34/lib",
+    "/usr/proj81/lib",
+    sep = ":"
+  )
+)
+
+# Specify additional proj path in which pkg-config should look for .pc files ----
+
+Sys.setenv("PKG_CONFIG_PATH" = "/usr/proj81/lib/pkgconfig")
+
+# Specify the path to GDAL data ----
+
+Sys.setenv("GDAL_DATA" = "/usr/gdal34/share/gdal")
+
+# Load geospatial libraries
+dyn.load("/usr/gdal34/lib/libgdal.so")
+dyn.load("/usr/geos310/lib64/libgeos_c.so", local = FALSE)
+```
+
+<details>
+
+<summary>This code will automatically add the required code</summary>
```

Comment:
```suggestion
<summary>This code will automatically add the required code (click to expand)</summary>
```

By default, it looks like this section is collapsed when viewing the .md. Maybe just add this to make absolutely clear to users?

Reply:
Yep it was deliberately collapsed but I agree your comment makes it clearer!